dada2_workflow.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: "DADA2 tutorial"
+output: html_notebook
+---
+
+Example of DADA2 tutorial can be found from [here](https://benjjneb.github.io/dada2/tutorial.html).
+This example include only the code. The pipelines ends to _TreeSummarizedExperiment_.
+
+```{r dada2_1}
+# GETTING READY
+
+library(dada2); packageVersion("dada2")
+```
+
+```{r dada2_2}
+path <- "~/Desktop/dada2_example/MiSeq_SOP" # CHANGE ME to the directory containing the fastq files after unzipping.
```

Comment:
Do not use local paths here. Just something like "data/MiSeq_SOP" or "data/dada2/MiSeq_SOP"